### ISSUE TITLE
Twitch Purple Screen Of Death

### ISSUE THREAD

**Issue by timothyerwin** [moonstar-x/react-twitch-embed#35]
I'm getting the Twitch purple screen when embedding...I tried adding the parent domains in the array but it doesn't help...anyone know if this is just Twitch showing it no matter what as seems to be indicated on other forums?

Any way to get rid of it?

**Comment by moonstar-x** [comment]
I tried accessing one of my sites that uses this package and it works completely fine.

What component are you using?

**Comment by moonstar-x** [comment]
Hmm, I've never seen this before...

I think it was worth mentioning that embed I tried was for a stream that wasn't currently online and it was from a phone browser with no adblock.

After digging for some info I came across [this article](https://gamerant.com/streamers-twitch-purple-screen-of-death/). I don't know how trustworthy it may be but it seems that this behavior has been introduced to lure viewers from outside Twitch into the main site. I'm sure that ad revenue is behind this decision.

I'm afraid it sounds that this is now the regular behavior and there's nothing that I can do to bypass this. 😢
